corehq/apps/app_manager/app_strings.py

```diff
@@ -166,6 +165,13 @@ def create_default_app_strings(self, app):
             if key not in messages:
                 messages[key] = u'Unable to find case referenced by auto-select case ID.'
 
+        key = 'case_autoload.{0}.property_missing'.format(AUTO_SELECT_LOCATION)
+        messages[key] = (u"This form requires access to the user's location, "
+                         "but none was found.")
+        key = 'case_autoload.{0}.case_missing'.format(AUTO_SELECT_LOCATION)
+        messages[key] = (u"This form requires the user's location to have a "
+                         "supply-point case.")
```

Comment:
I think the language here should be tweaked to better match the HQ UI. On HQ you select 'tracks stock' for a location type so I think this should say something like "This form requires a user's location to be configured for stock tracking."